tools/twix/src/change_buffer.rs

```diff
@@ -0,0 +1,144 @@
+use std::collections::VecDeque;
+
+use communication::{
+    client::{Communication, CyclerOutput, SubscriberMessage},
+    messages::Format,
+};
+use log::error;
+use serde_json::Value;
+use tokio::{
+    select, spawn,
+    sync::{mpsc, oneshot},
+};
+
+#[derive(Debug)]
+pub struct Change {
+    pub message_number: usize,
+    pub value: Value,
+}
+
+#[derive(Debug)]
+pub struct ChangeBufferUpdate {
+    pub updates: Vec<Change>,
+    pub message_count: usize,
+}
+
+pub enum Message {
+    GetAndReset {
+        response_sender: oneshot::Sender<Result<ChangeBufferUpdate, String>>,
```

Comment:
replace use by import

Reply:
I disagree. Just a few lines below this (l. 34), we use a `mpsc::Sender`. Having the channel type as a prefix here is the best way to distinguish between them IMO.